modules/local/macs2_consensus.nf

```diff
@@ -34,7 +23,7 @@ process MACS2_CONSENSUS {
 
     script: // This script is bundled with the pipeline, in nf-core/chipseq/bin/
     if (meta.multiple_groups || meta.replicates_exist) {
-        def prefix       = options.suffix ? "${meta.id}${options.suffix}.consensus_peaks" : "${meta.id}.consensus_peaks"
+        def prefix       = task.ext.prefix ? "${task.ext.prefix}.consensus_peaks": "${meta.id}.consensus_peaks"
```

Comment:
Maybe this should be set in `modules.config` instead? 
```suggestion
        def prefix = task.ext.prefix ?: "${meta.id}"
```

Reply:
Addressed in 38769b6be5eab09c037e45f6292daaba60b4db3f